api/api_full.go

```diff
@@ -885,6 +885,22 @@ type FullNode interface {
 
 	RaftState(ctx context.Context) (*RaftStateData, error) //perm:read
 	RaftLeader(ctx context.Context) (peer.ID, error)       //perm:read
+
+	// Actor events
+
+	// GetActorEvents returns all FEVM Actor events that match the given filter
+	// This is a request/response API. Please see the docs for `ActorEventFilter` for a detailed
+	// description of the filter options.
```

Comment:
```suggestion
	// GetActorEvents returns all FEVM Actor events that match the given filter.
	// This is a request/response API. Please see the docs for `ActorEventFilter` for a detailed
	// description of the filter options.
```

Reply:
@rjan90 is this an appropriate place to add "experimental"? Maybe like:

```suggestion
	// GetActorEvents returns all FEVM Actor events that match the given filter
	// This is a request/response API. Please see the docs for `ActorEventFilter` for a detailed
	// description of the filter options.
	// NOTE: This is an EXPERIMENTAL API and is subject to change in future releases.
```